frontend/src/Explorer/View/Common.purs

```diff
@@ -158,7 +160,10 @@ txBodyView (TxBodyViewProps props) =
             <<< map txFromView $ props ^. txbInputs
         , P.div
             [ P.className "to-hash-container bg-transaction-arrow" ]
-            <<< map txToView $ props ^. txbOutputs
+            [ P.div
+                [ P.className "to-hash-wrapper"]
+                <<< map txToView $ props ^. txbOutputs
```

Comment:
It (almost) is the same. The first one is using `lenses` and the second one `record accessor`. To be consistent in all code I would prefer to use the first approach (as long as we are using lenses).